### PR TITLE
1246/clutch

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1246 .

### CODE REVIEW

**Comment by commjoen** [comment]
@TheDauntless i just wanted to speed it up, so i prepared you a PR with some additions of mine, feel free to further refine/extend it and work on the feedback provided by @sushi2k

File: Document/0x06b-Basic-Security-Testing.md [hunk 1]
```diff
@@ -275,62 +275,136 @@ Save the IPA file locally with the following command:
...
+//                       Run path: @executable_path/Frameworks
+//                               = /Frameworks
+//         This file is encrypted:
+//                                   cryptid: 0x00000001
+//                                  cryptoff: 0x00004000
+//                                 cryptsize: 0x000fc000
+//
+```
+
+In order to retrieve the unencrypted version, we can use tools such as [frida-ios-dump](https://github.com/AloneMonkey/frida-ios-dump "Frida-iOS-dump") or [Clutch](https://github.com/KJCracks/Clutch "Clutch"). Both will extract the unencrypted version from memory while the application is running on the device. The stability of both Clutch and Frida can vary depending on your iOS version and Jailbreak method, so it's usefull to have multiple ways for extracting the binary. In general, all iOS versions lower than 12 should work with Clutch, while iOS 12+ should work with frida-ios-dump or modify Clutch as discussed later.
```
Comment (sushi2k) [inline Document/0x06b-Basic-Security-Testing.md]: usefull -> useful

File: Document/0x06b-Basic-Security-Testing.md [hunk 2]
```diff
@@ -275,62 +275,136 @@ Save the IPA file locally with the following command:
...
+
+struct CGPoint {
+    double _field1;
+    double _field2;
+};
+...
+
+```
+
+Note: when you use Clutch on iOS 12, please check [Clutch its Github issue 228](https://github.com/KJCracks/Clutch/issues/228 "Getting Clutch to run on iOS 12")
```
Comment (sushi2k) [inline Document/0x06b-Basic-Security-Testing.md]: remove "its"

File: Document/0x06b-Basic-Security-Testing.md [hunk 3]
```diff
@@ -275,62 +275,136 @@ Save the IPA file locally with the following command:
...
+
+###### Using Frida-ios-dump
+
+Frida-ios-dump requires Frida server running on your jailbroken device. It is basically using Frida scripts to dump the decrypted binary from memory onto a file.
+Note: before we get started, please note that the Frida-ios-dump tool is not always compatible with the latest version of Frida. Therefore: it might well be that you have to install an older version of Frida on your jailbroken device.
+First, make sure that the configuration in `dump.py` is set to either localhost with port 2222 when using iProxy, or to the actual ip-address and prot of the device from which you want to dump the binary. Next, change the username and password to the ones you use. Now you can safely use the tool to enumerate the apps installed:
```
Comment (sushi2k) [inline Document/0x06b-Basic-Security-Testing.md]: ip-address -> IP address 
prot -> port

File: Document/0x06b-Basic-Security-Testing.md [hunk 4]
```diff
@@ -275,62 +275,136 @@ Save the IPA file locally with the following command:
...
+ 685  Mail             com.apple.mobilemail
+ 834  Telegram         ph.telegra.Telegraph
+   -  Aandelen         com.apple.stocks
+   ...
+```
+
+and you can dump the binary you select:
+
+```shell
+python dump.py com.highaltitudehacks.DVIAswiftv2
```
Comment (sushi2k) [inline Document/0x06b-Basic-Security-Testing.md]: Add the "$" at the beginning

**Comment by OMTGreleaser** [comment]
Results for commit f82cb0f154cb5f8641d92c216c03f770131f98d9: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        6 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 910f9c44ea8e3df38e3c4576d8f2316ccfb02812: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        5 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**